server/src/internalClusterTest/java/org/opensearch/indices/replication/SegmentReplicationSuiteIT.java

```diff
@@ -13,15 +13,16 @@
 import org.opensearch.common.settings.Settings;
 import org.opensearch.indices.replication.common.ReplicationType;
 import org.opensearch.test.OpenSearchIntegTestCase;
+import org.opensearch.test.junit.annotations.TestLogging;
 import org.junit.Before;
 
 @OpenSearchIntegTestCase.ClusterScope(scope = OpenSearchIntegTestCase.Scope.SUITE, minNumDataNodes = 2)
+@TestLogging(reason = "Getting trace logs from replication package", value = "org.opensearch.indices.replication:TRACE")
```

Comment:
I am happy to merge a change that adds more logging so we can better root cause a source of failure; however, I don't think the others changes should be merged.

Reply:
+1 - @Rishikesh1159 mind making this update and we can add the test logging?